src/js/dropdown-menu/dropdown.underscore

```diff
@@ -9,10 +9,8 @@
 </a>
 <button type="button" class="menu-button button-more has-dropdown js-dropdown-button <% if (!button.icon) { %>default-icon<% } %>" aria-haspopup="true" aria-expanded="false" aria-controls="<%- menuId %>">
     <% if (button.icon) { %>
-        <span class="icon-fallback icon-fallback-img">
-            <span class="icon <%- button.icon %>" aria-hidden="true"></span>
-            <span class="text"><%- button.label %></span>
-        </span>
+        <span class="icon fa <%- button.icon %>" aria-hidden="true"></span>
+        <span class="sr-only"><%- button.label %></span>
     <% } else { %>
         <span class="sr-only"><%- button.label %></span>
     <% } %>
```

Comment:
This can be updated so if button.icon it adds the button, and it always adds the label element. i.e.
```
<% if (button.icon) { %>
    <span class="icon fa <%- button.icon %>" aria-hidden="true"></span>
<% } %>
<span class="sr-only"><%- button.label %></span>
```